Dockerfile

```
@@ -15,8 +15,8 @@ USER       nobody
 EXPOSE     9090
 VOLUME     [ "/prometheus" ]
 WORKDIR    /prometheus
-ENTRYPOINT [ "/bin/prometheus" ]
-CMD        [ "--config.file=/etc/prometheus/prometheus.yml", \
+ENTRYPOINT [ "/bin/prometheus", \
              "--storage.tsdb.path=/prometheus", \
              "--web.console.libraries=/usr/share/prometheus/console_libraries", \
-             "--web.console.templates=/usr/share/prometheus/consoles" ]
+             "--web.console.templates=/usr/share/prometheus/consoles"]
```

Style difference `"]` => `" ]`
```suggestion
             "--web.console.templates=/usr/share/prometheus/consoles" ]
```

---

good catch and applied suggestion!